middleware/getResults.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const async = require('async');
+const xmlParser = require('../lib/xmlParser');
+const utils = require('../lib/resultsUtils');
+const processResults = require('../lib/resultsProcessor.js');
+
+function getData(requestUrl, page, cb) {
```

Comment:
Are we adopting a convention of a small number of common abbreviations of which cb for callback will be one. I mention this as we discourage undue terseness in naming in out contribution notes.